homework/assignment_3/spark_connection.py

```diff
@@ -0,0 +1,57 @@
+# libraries
+import sys
+
+from pyspark.sql import SparkSession
+
+
+def main():
+    appName = "PySpark Connection"
+    master = "local"
+
+    # create spark session
+    spark = SparkSession.builder.appName(appName).master(master).getOrCreate()
+
+    # SQL query for 100 day rolling avg
+    sql = """
+        SELECT
+            bc.game_id,
+            SUM(bc.Hit) AS total_hits,
+            SUM(bc.atBat) AS total_atBats,
+            SUM(bc.Hit)/SUM(bc.atBat) AS batting_avg,
+            DATE(g.local_date) AS ora_date,
+            AVG(SUM(bc.Hit)/SUM(bc.atBat)) OVER (ORDER BY g.game_id, DATE(g.local_date)
```

Comment:
I think this is still the same issue where averaging the average will give you incorrect outputs